include/swift/AST/Module.h

```diff
@@ -203,6 +205,10 @@ class ModuleDecl : public DeclContext, public TypeDecl {
 
   SmallVector<FileUnit *, 2> Files;
 
+  friend OverlayFile;
```

Comment:
Is there a specific reason you separated out `class OverlayFile;` and `friend OverlayFile;` instead of having one `friend class OverlayFile;`? [I usually prefer the latter, I don't know if there is a reason to prefer the former.]

Reply:
Also, I can't quite tell where this `friend` relation is being used -- could you point that out?

Reply:
The need for friend might have been refactored away—I’m checking this.

Reply:
Yeah, I don't need it anymore. Good catch!

Reply:
Removed.